gsy_framework/sim_results/electric_blue/aggregate_results.py

```diff
@@ -0,0 +1,147 @@
+from collections import defaultdict
+from dataclasses import dataclass, field, fields
+from typing import Dict, List
+
+from pendulum import DateTime
+
+from gsy_framework.enums import AggregationResolution, AvailableMarketTypes
+
+# Used by forward markets; the following dictionary defines
+# what aggregations are needed for each market type.
+MARKET_RESOLUTIONS = {
+    AvailableMarketTypes.YEAR_FORWARD: [AggregationResolution.RES_1_MONTH],
+    AvailableMarketTypes.MONTH_FORWARD: [AggregationResolution.RES_1_DAY],
+    AvailableMarketTypes.WEEK_FORWARD: [AggregationResolution.RES_1_DAY]
+}
+
+
+@dataclass
+class ForwardDeviceStats:  # pylint: disable=too-many-instance-attributes
```

Comment:
Nothing to do here, but I have an almost-duplicate dataclass in gsy-web. I will import it in gsy-web from here if possible, once this PR is merged in order to not have duplicated code.

Reply:
You may also want to add some more properties to this class such as: `executed_trades`.